libphobos/libdruntime/Makefile.am

```diff
@@ -15,186 +15,201 @@
-	rm -f libgdruntime.a
-	rm -f libgdruntime_t.a
-
-check-local: unittest
-	./unittest
-
-# Work around what appears to be a GNU make bug handling MAKEFLAGS
-# values defined in terms of make variables, as is the case for CC and
-# friends when we are called from the top level Makefile.
-AM_MAKEFLAGS = \
-	"gdc_include_dir=$(gdc_include_dir)"
-
-# Subdir rules rely on $(FLAGS_TO_PASS)
-FLAGS_TO_PASS = $(AM_MAKEFLAGS)
+# Source file definitions. Boring stuff, auto-generated with
+# https://gist.github.com/jpf91/8ad1dbc9902d6ad876313f134c6527d1
+# Can't use wildcards here: https://www.gnu.org/software/automake/manual/html_node/Wildcards.html
+DRUNTIME_DSOURCES = core/atomic.d core/bitop.d core/checkedint.d core/cpuid.d core/demangle.d \
+	core/exception.d core/internal/convert.d core/internal/hash.d core/internal/traits.d \
+	core/math.d core/memory.d core/runtime.d core/simd.d core/stdc/complex.d core/stdc/config.d \
+	core/stdc/ctype.d core/stdc/errno.d core/stdc/fenv.d core/stdc/float_.d core/stdc/inttypes.d \
+	core/stdc/limits.d core/stdc/locale.d core/stdc/math.d core/stdc/signal.d core/stdc/stdarg.d \
+	core/stdc/stddef.d core/stdc/stdint.d core/stdc/stdio.d core/stdc/stdlib.d core/stdc/string.d \
+	core/stdc/tgmath.d core/stdc/time.d core/stdc/wchar_.d core/stdc/wctype.d core/sync/barrier.d \
+	core/sync/condition.d core/sync/config.d core/sync/exception.d core/sync/mutex.d \
+	core/sync/rwmutex.d core/sync/semaphore.d core/thread.d core/time.d core/vararg.d \
+	etc/linux/memoryerror.d gcc/atomics.d gcc/attribute.d gcc/backtrace.d gcc/builtins.d \
+	gcc/deh.d gcc/unwind/arm.d gcc/unwind/generic.d gcc/unwind/package.d gcc/unwind/pe.d \
+	object.d rt/aApply.d rt/aApplyR.d rt/aaA.d rt/adi.d rt/arrayassign.d rt/arraycast.d \
+	rt/arraycat.d rt/cast_.d rt/config.d rt/critical_.d rt/deh.d rt/dmain2.d rt/invariant.d \
+	rt/lifetime.d rt/memory.d rt/minfo.d rt/monitor_.d rt/obj.d rt/qsort.d rt/switch_.d \
+	rt/tlsgc.d rt/typeinfo/ti_AC.d rt/typeinfo/ti_Acdouble.d rt/typeinfo/ti_Acfloat.d \
+	rt/typeinfo/ti_Acreal.d rt/typeinfo/ti_Adouble.d rt/typeinfo/ti_Afloat.d rt/typeinfo/ti_Ag.d \
+	rt/typeinfo/ti_Aint.d rt/typeinfo/ti_Along.d rt/typeinfo/ti_Areal.d rt/typeinfo/ti_Ashort.d \
+	rt/typeinfo/ti_C.d rt/typeinfo/ti_byte.d rt/typeinfo/ti_cdouble.d rt/typeinfo/ti_cfloat.d \
+	rt/typeinfo/ti_char.d rt/typeinfo/ti_creal.d rt/typeinfo/ti_dchar.d rt/typeinfo/ti_delegate.d \
+	rt/typeinfo/ti_double.d rt/typeinfo/ti_float.d rt/typeinfo/ti_idouble.d rt/typeinfo/ti_ifloat.d \
+	rt/typeinfo/ti_int.d rt/typeinfo/ti_ireal.d rt/typeinfo/ti_long.d rt/typeinfo/ti_ptr.d \
+	rt/typeinfo/ti_real.d rt/typeinfo/ti_short.d rt/typeinfo/ti_ubyte.d rt/typeinfo/ti_uint.d \
+	rt/typeinfo/ti_ulong.d rt/typeinfo/ti_ushort.d rt/typeinfo/ti_void.d rt/typeinfo/ti_wchar.d \
+	rt/util/array.d rt/util/container/array.d rt/util/container/common.d rt/util/container/hashtab.d \
+	rt/util/container/treap.d rt/util/hash.d rt/util/random.d rt/util/string.d rt/util/typeinfo.d \
```

Comment:
I would like to keep to a 80 character limit per line.  Would that be a problem?